test/unit/lib/censorDeath.js

```diff
@@ -0,0 +1,268 @@
+    ageAtDeath: null,
+    birthplace: null,
+    deathplace: null,
+    sex: null,
+    address: null,
+    occupation: null,
+    retired: null,
+    maidenSurname: null,
+    causeOfDeath: null,
+    certifiedBy: null,
+    relationshipToPartner: null,
+    aliases: []
+  },
+  informant: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: null,
+    signature: null
+  },
+  registrar: {
+    signature: null,
+    designation: null,
+    subdistrict: null,
+    district: null,
+    administrativeArea: null
+  },
+  coroner: {
+    name: null,
+    designation: null,
+    area: null
+  },
+  partner: {
+    name: null,
+    occupation: null,
+    retired: null
+  },
+  father: {
+    name: null,
+    occupation: null
+  },
+  mother: {
+    name: null,
+    occupation: null
+  },
+  status: {
+    blocked: true,
+    correction: null,
+    marginalNote: null,
+    onAuthorityOfRegistrarGeneral: null
+  },
+  previousRegistration: {},
+  nextRegistration: {}
+}
+
+const redactedDataNotBlocked = {
+  id: 1000,
+  date: "2020-09-23",
+  entryNumber: null,
+  inquestDate: null,
+  deceased: {
+    prefix: null,
+    forenames: "Forename1 Forname2",
+    surname: "SURNAME",
+    suffix: null,
+    ageAtDeath: null,
+    birthplace: null,
+    deathplace: null,
+    dateOfBirth: "2020-09-23",
+    dateOfDeath: "2020-09-23",
+    dateOfDeathQualifier: "On or about",
+    sex: "Indeterminate",
+    occupation: null,
+    retired: null,
+    maidenSurname: null,
+    causeOfDeath: null,
+    certifiedBy: null,
+    relationshipToPartner: null,
+    address: "Current Address",
+    aliases: [{
+      type: 'Stage Name',
+      "prefix": "The Artist Formally Known As...",
+      "forenames": "Prince Rogers",
+      "surname": "Prince Rogers Nelson",
+      "suffix": "Unpronounceable Symbol"
+    }, {
+      type: 'Undercover',
+      "prefix": "Agent",
+      "forenames": "Alec",
+      "surname": "Trevelyan",
+      "suffix": "006"
+    }]
+  },
+  informant: {
+    forenames: null,
+    surname: null,
+    address: null,
+    qualification: null,
+    signature: null
+  },
+  registrar: {
+    signature: null,
+    designation: null,
+    subdistrict: "Subdistrict town",
+    district: "District city",
+    administrativeArea: "Adminshire"
+  },
+  coroner: {
+    name: null,
+    designation: null,
+    area: null
+  },
+  partner: {
+    name: null,
+    occupation: null,
+    retired: null
+  },
+  father: {
+    name: null,
+    occupation: null
+  },
+  mother: {
+    name: null,
+    occupation: null
+  },
+  status: {
+    blocked: false,
+    correction: null,
+    marginalNote: null,
+    onAuthorityOfRegistrarGeneral: null
+  },
+  previousRegistration: {},
+  nextRegistration: {}
+}
+
+describe('lib/censorDeath.js', () => {
+  describe('censorDeath function', () => {
+    it('should be a function', () => {
+      expect(censorDeath).to.be.a('function')
+    })
+    describe('record not blocked', () => {
```

Comment:
Better to write some nicer English here. Perhaps `'when the record is NOT blocked'`.

Reply:
The reason being that we want some poor future programmer to be able to understand what has gone wrong as soon as possible.